<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirPalClass</class>
 <widget class="QMainWindow" name="AirPalClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AirPal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>220</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>My Bookings</string>
    </property>
   </widget>
   <widget class="QLabel" name="myBookingsSubtitle">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>450</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica 45 Light</family>
     </font>
    </property>
    <property name="text">
     <string>You have no upcoming flights!</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>791</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>741</width>
       <height>121</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>701</width>
        <height>121</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>701</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>701</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Resources/airplane logo.jpg</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>140</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica 45 Light</family>
      </font>
     </property>
     <property name="text">
      <string>SEARCH</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="searchResultsList">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>751</width>
       <height>311</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="addTransitsbutton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="costBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>140</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
     <property name="singleStep">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>140</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2019</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="minimumDate">
      <date>
       <year>2019</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="nextButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>490</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QComboBox" name="pageLengthBox">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>500</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>500</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Show first</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>500</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>results</string>
     </property>
    </widget>
    <widget class="QLabel" name="searchResultsSubtitle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>500</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="airlineSelect">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>140</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Any Airline</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ANA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Emirates</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Qatar</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="efficiencyBox">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>140</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Cost</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Time</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="searchQuery">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="searchQuery_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="filterWarning">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>500</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="bookingsList">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>250</y>
      <width>161</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QLabel" name="userLoad">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>70</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica 45 Light</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;No user loaded&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway Medium</family>
    </font>
   </property>
   <widget class="QMenu" name="menuhello_world">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Terminal"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionLicense"/>
   </widget>
   <addaction name="menuhello_world"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Terminal">
   <property name="text">
    <string>Open Terminal...</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway Medium</family>
    </font>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit AirPal</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway Medium</family>
    </font>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway Medium</family>
    </font>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway Medium</family>
    </font>
   </property>
  </action>
  <action name="actionUser">
   <property name="text">
    <string>User</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway Medium</family>
    </font>
   </property>
  </action>
  <action name="actionAdmin">
   <property name="text">
    <string>Admin</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway Medium</family>
    </font>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway Medium</family>
    </font>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway Medium</family>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AirPal.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bookingsList</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>AirPalClass</receiver>
   <slot>openThisTicket()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>AirPalClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Terminal</sender>
   <signal>triggered()</signal>
   <receiver>AirPalClass</receiver>
   <slot>openFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>AirPalClass</receiver>
   <slot>saveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_As</sender>
   <signal>triggered()</signal>
   <receiver>AirPalClass</receiver>
   <slot>saveAsFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchResultsList</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>AirPalClass</receiver>
   <slot>openThisQuery()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>myBookingsSubtitle</sender>
   <signal>windowTitleChanged(QString)</signal>
   <receiver>AirPalClass</receiver>
   <slot>triggerIsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>AirPalClass</receiver>
   <slot>nextPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDocumentation</sender>
   <signal>triggered()</signal>
   <receiver>AirPalClass</receiver>
   <slot>openAction_Documentation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchQuery</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AirPalClass</receiver>
   <slot>analyzeText1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchQuery_2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AirPalClass</receiver>
   <slot>analyzeText1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openThisTicket()</slot>
  <slot>openFile()</slot>
  <slot>saveAsFile()</slot>
  <slot>saveFile()</slot>
  <slot>openThisQuery()</slot>
  <slot>triggerIsChanged()</slot>
  <slot>nextPage()</slot>
  <slot>openAction_Documentation()</slot>
  <slot>analyzeText1()</slot>
  <slot>analyzeText2()</slot>
  <slot>changeTransitParams_DIRECT()</slot>
  <slot>changeTransitParams_INDIRECT()</slot>
 </slots>
</ui>
